.github/workflows/openstack-cloudlinux-7.yml

```diff
@@ -175,7 +175,7 @@ jobs:
               chmod -v +x /opt/${REPODIR}/t/integration/setup
               /opt/${REPODIR}/t/integration/setup
 
-    start_elevate:
+    prep_elevate:
```

Comment:
FIX: Move this to its own PR.  RE-1028 was created to handle the post_snapshot issues.  Also, I believe that the c7 run should have this included to it as well, correct?